code/signup.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Sign Up</title>
+</head>
+<body>
+
+    <section class="form-container">
+        
+    <form action="https://httpbin.org/anything" method="POST">
+        <a href="index.html" class="back">X</a>
+        <div class="welcome">
+            <h3>CREATE ACCOUNT</h3>
+        </div>
+        
+        <div class="textboxes">
+                
+            <label for="name"></label>
+            <input type="text" required id="name" name="user_name" placeholder="Full name">
+            <br>
```

Comment:
We shouldn't use `<br />` tags - more details in [Week 2 Master document](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)